src/wasm/wasm.cpp

```diff
@@ -198,7 +198,7 @@ const char* getExpressionName(Expression* curr) {
   WASM_UNREACHABLE("invalid expr id");
 }
 
-Literal getLiteralFromConstExpression(Expression* curr) {
+Literal getSingleLiteralFromConstExpression(Expression* curr) {
```

Comment:
Is the name here meant to avoid confusion as `Literal` and `Literals` are too close? (I slightly prefer `Literal` over `SingleLiteral` despite that risk.)

Reply:
Yes, but making the difference more obvious also serves the purpose of calling out that this function explicitly does not work for tuples. That could be useful to prompt us in the future to consider whether tuples need to be handled or not.

Reply:
Fair point. It seems like Literal/Literals is already making that clear, though? If we do want to stress this, it seems like the other way might be better, as multis are rarer, so `getLiteral` / `getMultiLiteral` perhaps? But I don't feel strongly here.

Reply:
I think @aheejin had a preference for `getSingle...`, which is how we ended up with `Flow::getSingleValue`. Also, multivalue flows are definitely rarer than single value flows, but I expect that most code will need to handle both and knowing that a value is not a tuple will be the exception.